Classes/Neos/Neos/Ui/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -103,20 +105,6 @@ public function renderNode(NodeInterface $node, ControllerContext $controllerCon
                 $nodeInfo['previewUri'] = $this->uri($nodeInLiveWorkspace, $controllerContext);
             }
         }
-
-        // child nodes for document tree, respecting the `baseNodeType` filter
-        $documentChildNodes = $node->getChildNodes($baseNodeType);
```

Comment:
@dimaip does the BaseNodeType filter in the tree still work? :-) Just asking

Reply:
@skurfuerst yes, absolutely, just tested it with `baseNodeType: 'Neos.Neos:Document,!Neos.Neos:Shortcut'`:

![image](https://user-images.githubusercontent.com/837032/36370975-c6f1acac-1571-11e8-8e18-819daf3fd9a8.png)
`